### PR TITLE
Add property in ADF ExecuteSSISPackage activity

### PR DESCRIPTION
**Pull request by qianwens**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by qianwens** [comment]
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/2716

**Comment by qianwens** [comment]
@azuresdkci it seems this build is hang in the test step for more than 20 hours, can we restart this build?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please create a `generate.ps1` similar to [this](https://github.com/qianwens/azure-sdk-for-net/blob/b0083a8a7108abbd82726951e0bc32b52440a52c/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.
There are additive changes to the sdk, please rev the csproj version

**Comment by qianwens** [comment]
@dsgouda there is a generate.cmd file in DataFactory folder which call generate.ps1 as the example. @hvermis should I update the version to 0.7.0-preview in this change?

**Comment by dsgouda** [comment]
@qianwens 
I did not understand the first question
Since these are additive changes you can simply bump it to `0.6.1-preview`

**Comment by dsgouda** [comment]
@azuresdkci retest this please

**Comment by hvermis** [comment]
@qianwens , please add the description of your changes on changelog.md under current version. Please add this to the top of that file and add your own change description below it:
## Current version

### Feature Additions

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
 		<Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-		<Version>0.6.0-preview</Version>
+		<Version>0.6.1-preview</Version>
```
Comment (hvermis) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Please revert the changes in this file too. I will update all needed files before publishing the nuget.

**Review by hvermis — CHANGES_REQUESTED** [review]

**Review by hvermis — APPROVED** [review]

**Comment by qianwens** [comment]
@dsgouda I mean there is a generate.cmd file in DataFactory folder already so I think I do not need to create it as you requested. Hermine will update the version before publishing the nuget.

**Review by dsgouda — APPROVED** [review]
LGTM with the understanding that the package version will be updated prior to publishing

**Pull request merged**